src/static_api.rs

```diff
@@ -98,6 +104,7 @@ impl<'a> Generator<'a> {
                     })
                     .collect(),
                 branch_protections,
+                archived,
```

Comment:
Hmmm... Do we want to include archived repos in the repos endpoint or continue to only show current repos and have a dedicared `archived_repos` endpoint?

I think I would prefer the dedicated endpoint approach.

Reply:
Hmm, we don't even have an endpoint for archived teams. So maybe we should not even expose the archived repos as an endpoint?

Reply:
Not including the archived status may cause issues for sync-team, since it won't know the difference between a normal repo, an archived repo, and a repo that is not yet tracked.

(IMHO, I think a flag is sufficient since this only really matters for sync-team, and it is also similar to the way GitHub's API works.)

Reply:
Uhh, of course, we need to expose that information, that's the whole point of this PR, silly me :) The flag will definitely make both `team` and `sync-team` much simpler.

Reply:
I guess that I don't consider it to be that fundamental :thinking: For example, we might also want to manage private repos in the future, should these also have a separate endpoint in your opinion?

Reply:
That raises a good point. Should archived repos be interactable and have any properties set at all? E.g., what happens if you change a branch protection for an archived repo? Should that do something? (Will that unarchive the repo automatically?).

Reply:
I was definitely working under the assumption that archived repos are in a read-only state (more or less by definition), and in order to change them, you would need to unarchive them.

Reply:
I think it is fine if sync-team makes changes to archived repos. It won't unarchive them, and the API seems to be fine with making changes.

Reply:
To clarify: I don't think it matters much. There might be small things, for example the description. Someone may want to change that, and I think it would be kinda fussy to have to unarchive, change, and then re-archive. But I don't expect that to happen much, so I think it probably doesn't matter.